drivers/serial/uart_mcux_lpuart.c

```diff
@@ -1092,13 +1092,17 @@ static int mcux_lpuart_configure_init(const struct device *dev, const struct uar
 
 	LPUART_Init(config->base, &uart_config, clock_freq);
 
+#if defined(FSL_FEATURE_LPUART_HAS_MODEM_SUPPORT) && \
+	FSL_FEATURE_LPUART_HAS_MODEM_SUPPORT
```

Comment:
the ci check wants this to be on one line

Reply:
CI check does not seem to fail. There has been one more instance of the same line in the file before my change already. Unified style of the file would be nice. Can I leave the added lines as is, or do you prefer to change all occurrences to the one line format?

Reply:
This commit should come before the commits adding support for the MCXC242, if that is the SOC causing this build error

Reply:
Moved the commit.